src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarCommercial.test.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  ArtworkBuyNow,
+  ArtworkBuyNowMakeOffer,
+  ArtworkMakeOffer,
+  ArtworkSold,
+} from "Apps/__tests__/Fixtures/Artwork/ArtworkSidebar/ArtworkSidebarCommercial"
+import { ArtworkSidebarCommercialFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial"
+import { renderRelayTree } from "DevTools"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+describe("ArtworkSidebarCommercial", () => {
+  const getWrapper = async response => {
+    return await renderRelayTree({
+      Component: ArtworkSidebarCommercialFragmentContainer,
+      query: graphql`
+        query ArtworkSidebarCommercial_Test_Query {
+          artwork(id: "commercial_artwork") {
+            ...ArtworkSidebarCommercial_artwork
+          }
+        }
+      `,
+      mockResolvers: {
+        Artwork: () => response,
+      },
+    })
+  }
+
+  let artwork
+
+  it("displays artwork enrolled in Buy Now", async () => {
+    artwork = Object.assign({}, ArtworkBuyNow)
+
+    const wrapper = await getWrapper(artwork)
+
+    expect(wrapper.text()).toContain(artwork.sale_message)
```

Comment:
should this be checking for the 'Buy Now' button/label, similar to below? (`sale_message` is something else and is displayed on the sidebar too).

Reply:
Yep! It certainly should be. Will follow-up on this. Thanks!